content/blog/2023/03/10/2023-03-10-jenkins-newsletter.adoc

```diff
@@ -0,0 +1,149 @@
+For all of the work and energy you're putting into the Jenkins project, we extend our deepest gratitude.
+
+[[Governance]]
+image:/images/post-images/2023/01/12/jenkins-newsletter/governance.png[Governance Update]
+
+Contributed by: author:markewaite[Mark Waite]
+
+The Jenkins governance board met once in February, resolved several action items, and noted the progress on projects with sponsors like JFrog and Atlassian.
+We're sincerely grateful for the sponsorships provided by those generous companies and many other companies.
+
+[[platform]]
+image:/images/post-images/2023/01/12/jenkins-newsletter/platform-modernization.png[Platform Modernization Update]
+
+Contributed by: author:gounthar[Bruno Verachten]
+
+Ongoing work:
+
+ - Centos `7` Jenkins Controller Docker Image proposal: deprecate the centos images.
+We'll need an announcement and a https://github.com/jenkinsci/jep[JEP].
+We'll also release a merged version of the `centos` and `centos7` image as the very last centos `7` Docker image.
+
+Docker images:
+
+* Latest updates on the agent images:
+** ssh-agent: bump Git version on Windows to 2.39.2.windows.1 (https://github.com/jenkinsci/docker-ssh-agent/pull/209[#209]) @github-actions
+** docker-agent: bump Git version on Windows to 2.39.2.windows.1 (https://github.com/jenkinsci/docker-agent/pull/376[#376]) @github-actions
+** inbound agent:
+*** bump the parent image jenkins/agent version to 3107.v665000b_51092-4 (https://github.com/jenkinsci/docker-inbound-agent/pull/331[#331]) @github-actions
+*** bump the parent image jenkins/agent version to 3107.v665000b_51092-3 (https://github.com/jenkinsci/docker-inbound-agent/pull/330[#330]) @github-actions
+*** bump updatecli/updatecli-action from 2.19.0 to 2.20.1 (https://github.com/jenkinsci/docker-inbound-agent/pull/329[#329]) https://github.com/dependabot[@dependabot]
+*** Windows controller image is not updated as often as the rest. It's been more than one year without any update. We may simply drop it.
+
+* Debian 12 (“bookworm”) will not deliver OpenJDK 11
+** The end of life date for Debian's openJDK11 won't happen until 2026 or 2027.
+** The documentation will be updated when it goes out, so that we describe the use of Jenkins with openJDK17.
+
+New platforms:
+
+* RISC-V support is far from official for Jenkins, but tests are progressing.
+
+[[modern-ui]]
+image:/images/post-images/2023/01/12/jenkins-newsletter/ui_ux.png[User Experience Update]
+
+Contributed by: author:markewaite[Mark Waite]
+
+User experience improvements continued to arrive in February thanks to contributions from author:janfaracik[Jan Faracik], author:notmyfault[Alexander Brandes], author:timja[Tim Jacomb], https://github.com/mawinter69[Markus Winter], and others.
+Look for the improvements in recent weekly releases and in the upcoming Jenkins `2.387.1` LTS release.
```

Comment:
```suggestion
Look for the improvements in recent weekly releases and in the new Jenkins `2.387.1` LTS release.
```
update wording since 2.387.1 has released today